openshift-api-for-data-protection-operator/base/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    kubernetes.io/metadata.name: openshift-adp
+  name: openshift-adp
+spec:
```

Comment:
Please remove the "spec" stanza, it's not required.

Reply:
Done